SCAutolib/utils.py

```diff
@@ -60,25 +60,29 @@ def _gen_private_key(key_path: Path):
 
 def _get_os_version():
     """
-    Find Linux version. Available version: RHEL 8, RHEL 9, Fedora.
+    Find Linux version. Available version: RHEL 8, RHEL 9, RHEL 10, Fedora.
     :return: Enum with OS version
     """
     with open('/etc/redhat-release', "r") as f:
```

Comment:
Should we use some standard way of detecting the os version instead of this, such as distro module (we should double-check the RHEL 10 is already detected correctly though)?

https://pypi.org/project/distro/

Or just throw away this part and use it directly?

Reply:
I tested it and I like it, not a big fan of adding more dependencies but it will make our lives much easier in the long run. Although it is a big change I would prefer to have a separate PR for that after this one.